### PR TITLE
C basics

### PR DESCRIPTION
**Pull request by JaiVallabh**

### CODE REVIEW

File: swap.cpp [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+OUTPUT:  Swap the values of a, b.
+
+ERROR CASES:
+
+NOTES:
+*/
+
+void swap(int *a, int *b)
+{
+	int c;
```
Comment (harinarayana1997) [inline swap.cpp]: Name is as temp or something, what's c?

Comment (JaiVallabh) [reply swap.cpp]: ok

- [ ] 

- [ ]

**Comment by Abhivardhan** [comment]
Naming variables should represent their purpose. Take care of them properly

File: nthNumber.cpp [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+				{
+					flag = 0;
+					break;      //Loop will terminate if p is not prime
+				}
+			}
+			if (flag == 1)
+				i++;
+			p++;
+		}
+	}return (p-1);
```
Comment (Abhivardhan) [inline nthNumber.cpp]: write return in next line

File: nthNumber.cpp [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+		}
+		return num3;
+	}
+}
+
+int nthPrime(int num)
+{
+	int  i, p, count, flag;
+	p = 2;
```
Comment (Abhivardhan) [inline nthNumber.cpp]: rename p

File: isPerfect.cpp [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+	int i,j,sum;
+	for (i = 0; i < length; i++)
+	{
+		 sum = 0;
+		for (j = 1; j<input[i]; j++)
+		{
+			if (input[i] % j == 0)
+				sum += j;
+		}
+		if (input[i] == sum)
```
Comment (Abhivardhan) [inline isPerfect.cpp]: use ternary operator. if checks are expensive

**Pull request merged**